packages/teleport/src/Audit/Audit.story.tsx

```diff
@@ -28,11 +29,22 @@ export default {
 export const Loaded = () => {
   const ctx = new Context();
   ctx.auditService.fetchEvents = () =>
-    Promise.resolve({ events, startKey: '' });
+    Promise.resolve({ events: events.slice(0, 20), startKey: '' });
```

Comment:
this can break the snapshot b/c it relies on the events order, so lets make a `eventsSample` in our fixtures, where we make like 4 events, and use that here. Lets also use this sample for the `LoadedFetchMore`

To make it easier on folks who add events for us, we can place a comment for `export const events = [`, that states that this list contains all known events